src/pages/send-bulk-message.js

```diff
@@ -0,0 +1,216 @@
+import React from 'react';
+import Layout from '../components/layout';
+import Link from 'next/link';
+import { useState, useEffect } from 'react';
+import { Button, Checkbox, Dropdown } from '../components/Form';
+import { CallbackGateway } from '../gateways/callback';
+import { useRouter } from 'next/router';
+import {PRE_CALL_MESSAGE_TEMPLATE} from '../helpers/constants';
+import { GovNotifyGateway  } from "../gateways/gov-notify";
+import { unsafeExtractUser } from "../helpers/auth";
+import { callTypes } from "../helpers/constants";
+
+export default function AssignCallsPage() {
+    const router = useRouter();
+		const user = unsafeExtractUser()
+    const [assignedCallbacks ,setAssignedCallbacks]  = useState(null)
+    const [unAssignedCallbacks ,setUnAssignedCallbacks]  = useState(null)
+    const [previewTemplate, setTemplatePreview] = useState([])
+    const [unassigned, setUnassigned] = useState({
+      value:false,
+      sendSms:false
+    })
+    const [assigned, setAssigned] = useState({
+      value:false,
+      sendSms:false
+    })
+    const [helpType, setHelpTpye] = useState(null)
+    const [callbacks, setCallbacks] = useState(null)
+    const [dropdownItems, setDropDown] =  useState([""])
+    const [errorsExist, setErrorsExist] = useState(false)
+    const getCallbacks = async () => {
+      const callbackGateway = new CallbackGateway();
+      let callbacks = await callbackGateway.getCallback({});
+      setCallbacks(callbacks)
+    }
+
+    const getPreview = async () => {
+      let govNotifyGateway = new GovNotifyGateway
+      let response = await govNotifyGateway.getTemplatePreview(PRE_CALL_MESSAGE_TEMPLATE)
+      setTemplatePreview(response.body)
+    }
+
+    useEffect(()=> {
+      getPreview()
+    }, [])
+    useEffect(()=> {
+      getCallbacks()
+    }, [])
+    useEffect(()=>{
+      setDropDown(["Please choose"].concat(callTypes))
+    }, [])
+    
+
+    const handleReview = async (event) => {
```

Comment:
I'm not clear why this is called handleReview?

Reply:
yess, this button was previously called review, changed it to handleSend